packages/components/src/ListView/Items/Item/Item.component.js

```diff
@@ -42,6 +44,17 @@ class Item extends Component {
 			{ 'switch-nested': children },
 			{ switch: isSwitchBox },
 		);
+		const checked = !!item.checked;
```

Comment:
Default props could be used to avoid bang operator

Reply:
In fact we don't even need the default props. I just extracted the checked value that is passed in the input props, but even with undefined it's ok. I remove that